### PR TITLE
Rk unsw end survey inapp browser

### PR DESCRIPTION
**Pull request by atton16**
- Add End Survey
- Remove unwanted buttons in the settings page
  - Logout
  - Tracking
  - Medium Accuracy

Screenshots:
Screen Shot 2563-03-12 at 20 45 47
Screen Shot 2563-03-12 at 20 45 41

### CODE REVIEW

**Comment by asiripanich** [comment]
@shankari We are removing the logout button so that our users to make sure our users won't accidentally quit the survey midway. We will advice them to delete the app once they fill the post-experience survey.

**Comment by asiripanich** [comment]
@shankari Can you approve this as well?

File: www/js/control/general-settings.js [hunk 1]
```diff
@@ -415,6 +416,15 @@ angular.module('emission.main.control',['emission.services',
         EnketoSurveyLaunch.launch($scope, 'UserProfile');
     };
 
+    $scope.launchEndSurvey = function() {
+        CommHelper.getUser().then(function(profile) {
+            const uuid = profile && profile.user_id && profile.user_id['$uuid'] ? profile.user_id['$uuid'] : 'undefined';
+            const fieldPath = '/aDHunALXzzHXAUMrvkvRT9/_user_id';
+            const returnURL = 'https://reward.amarin.dev';
+            $cordovaInAppBrowser.open(`https://ee.kobotoolbox.org/single/::T9bEosso?d[${fieldPath}]=${uuid}&returnURL=${returnURL}`, '_blank');
```
Comment (shankari) [inline www/js/control/general-settings.js]: so for the end survey you decided to just use the online kobotoolbox and not embed it into the app?

Comment (shankari) [reply www/js/control/general-settings.js]: I have no problems with this; just wanted to check

Comment (asiripanich) [reply www/js/control/general-settings.js]: Yes, due to the time constraint that we are facing.

**Comment by asiripanich** [comment]
@shankari press build? 😬

**Comment by shankari** [comment]
wait, you want me to deploy this? I though you were only deploying to the `rciti1`, `rciti2`.... channels now

**Comment by asiripanich** [comment]
It is for testing. If works on rk-unsw then we gonna deploy to all other branches.

**Comment by shankari** [comment]
ok gimme 10 mins

**Comment by asiripanich** [comment]
Thank you :)

**Comment by shankari** [comment]
deployed, should be live in ~10 mins

**Pull request merged**